### PR TITLE
let test cases pass in window environment

### PR DESCRIPTION
**Pull request by p2227**

### CODE REVIEW

File: test/error.js [hunk 1]
```diff
@@ -8,14 +8,15 @@ const tman = require('tman')
...
+const path = require('path')
 
 tman.suite('error', function () {
   tman.it('new QuicError(0)', function () {
     let err = new QuicError(0)
     assert.strictEqual(err.code, 0)
     assert.strictEqual(err.name, 'NO_ERROR')
     assert.strictEqual(err.message, '')
-    assert.ok(err.stack.includes('test/error.js'))
+    assert.ok(err.stack.includes(path.normalize('test/error.js')))
```
Comment (zensh) [inline test/error.js]: I think it does not work for windows. We should use `path.join('test', 'error.js')`

Comment (p2227) [inline test/error.js]: aha, your opinion is better. Accept and change yourself or send another PR?
